Playwright Test: Add Tools -> Promote Blaze to Playwright Test

alisterscott opened the pull request:
## Proposed Changes

* Converts the existing e2e test for Blaze to Playwright Test

## Why are these changes being made?

* e2e Test Improvements for wp-calypso

## Testing Instructions

* `yarn test:pw ./specs/tools/advertising__promote.spec.ts --repeat-each=10`

alisterscott committed:
Add promote blaze spec

alisterscott committed:
Delete promote blaze from old framework

alisterscott committed:
Add 30s timeout for campaign landing page as this can take some time to load!

alisterscott committed:
Merge branch 'trunk' into add/e2e-test-tools-promote-blaze

alisterscott committed:
remove wait for this is covered before calling this method

adimoldovan commented on test/e2e/specs/tools/advertising__promote.spec.ts:
```diff
@@ -0,0 +1,74 @@
+import { expect, tags, test } from '../../lib/pw-base';
+import { TEST_IMAGE_PATH } from '../constants';
+
+test.describe( 'Advertising: Promote', { tag: [ tags.CALYPSO_PR ] }, () => {
```
```suggestion
test.describe( 'Advertising: Promote', { tag: [ tags.CALYPSO_PR, tags.JETPACK_WPCOM_INTEGRATION ] }, () => {
```

adimoldovan reviewed (changes requested):
Let's not remove the old test yet, the group is still in use in TC to test Jetpack deployments: https://github.com/Automattic/wp-calypso/blob/cb3d502afeaccb382597326ea6fc6a4dbb1007e7/.teamcity/_self/projects/WPComTests.kt#L201. But let's remove the calypso_pr group instead, so it doesn't still run on the old Jest configuration.

alisterscott committed:
Revert "Delete promote blaze from old framework"

This reverts commit 402676cc5ff1fdb29c16160677ba99767d040191.

alisterscott committed:
Added tags.JETPACK_WPCOM_INTEGRATION to advertising__promote.spec.ts

alisterscott committed:
Merge branch 'trunk' into add/e2e-test-tools-promote-blaze

alisterscott committed:
Remove calypso-pr from jest playwright test

alisterscott commented:
Thanks @adimoldovan 

I've:

* Kept `test/e2e/specs/tools/advertising__promote.ts` but removed `@group calypso-pr` from it
* Added `tags.JETPACK_WPCOM_INTEGRATION` to `test/e2e/specs/tools/advertising__promote.spec.ts`
* Updated from trunk

Would you mind having a look again when you have a chance?
